### PR TITLE
Kubectl annotate implementation

### PR DESCRIPTION
**Pull request by yue9944882**

### CODE REVIEW

File: util/src/main/java/io/kubernetes/client/util/annotations/Annotations.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  /**
+   * Adds one annotation to the object.
+   *
+   * @param kubernetesObject the kubernetes object
+   * @param annotation the annotation key
+   * @param value the annotations value
+   */
+  public static void addAnnotations(
+      KubernetesObject kubernetesObject, String annotation, String value) {
+    Map<String, String> mergingAnnotations = new HashMap<>();
```
Comment (brendandburns) [inline util/src/main/java/io/kubernetes/client/util/annotations/Annotations.java:30]: Replace these 3 lines with:

`addAnnotations(kubernetesObject, Maps.newHashMap(annotation, value));`

**Comment by brendandburns** [comment]
One really minor style comment then LGTM.

File: util/src/main/java/io/kubernetes/client/util/annotations/Annotations.java (hunk 1 above)
Comment (yue9944882) [reply util/src/main/java/io/kubernetes/client/util/annotations/Annotations.java]: i squashed the lines to:

> addAnnotations(kubernetesObject, ImmutableMap.of(annotation, value));

**Pull request merged**